FSW/src/headers/timing.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef TIMING_H
+#define TIMING_H 
+/* Function Declarations for Timing module
+ * (class definitions must be included with higher 
+ *   priority than config.hpp, but this file must be 
+ *   included after config.hpp to use BUFFERSIZE) */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+#include "config.hpp"
+#include "../headers/eventUtil.hpp"
+#include "../headers/comUtil.hpp"
+
+/* - - - - - - Enums - - - - - - - */
+enum Mode // all payload science modes
+{
+    SAFE_MODE,      // KEEP FIRST IN ENUM, disable all operations
+
+    // add new modes here
+    STANDBY_MODE,   // waiting for wakeup signal from bus
+    SUNSET_MODE,    // gathering data, waiting for phododiode threshold drop
+    PRE_SUNRISE_MODE, // gathering data, waiting for photodiode threshold rise
+    SUNRISE_MODE,   // gathering data on timer (returns to standby)
+
+    // end of list
+    // TODO: Do we actually need this check? Maybe a bit flip could lead to invalid states being returned? 

```

Comment:
Was this TODO discussed, I am curious as well.

Reply:
It has not been. I am just going to keep the check in because it isn't too hard, but I cannot really envision a scenario where the mode is set to an extraneous mode since we only set it with enum variables and are not setting it with manual integers